ods_ci/tests/Tests/0600__distributed_workloads/0602__training/test-run-training-stack-tests.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Documentation     Training operator E2E tests - https://github.com/opendatahub-io/distributed-workloads/tree/main/tests/kfto/core
+Suite Setup       Prepare Training Operator E2E Core Test Suite
+Suite Teardown    Teardown Training Operator E2E Core Test Suite
+Library           OperatingSystem
+Library           Process
+Resource          ../../../../tasks/Resources/RHODS_OLM/install/oc_install.robot
+Resource          ../../../../tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource
+
+
+*** Variables ***
+${CUDA_TRAINING_IMAGE}            quay.io/modh/training@sha256:b98e373a972ff6f896a9dc054d56920e915675339c02ea7fa123e0f4bbef4d74 
```

Comment:
Can you move the images to `DistributedWorkloads.resource`, so we have all images on one place?